qna/DE-action-start-de.md

```diff
@@ -1,33 +1,33 @@
 ## Question
-How is the start time of the action determined for the german market?
+How is the start time of a transaction determined for the German market?
 
 ## Question
-How can I set the start time of an action for the german market?
+How can I set the start time of a transaction for the German market?
 
 ## Question
-Which start times of an action must be printed on the receipt in germany?
+Which start times of a transaction must be printed on the receipt in Germany?
 
 ## Metadata tags
 lang-en, market-de, middleware
 
 ## Answer
-On the receipt 3 datetimes, in special cases even 4 datetimes must be printed:
+The receipt must contain three date/time pairs, in cases of long lasting transactions even four:
 
-1. Date of receipt issuance/creation (DE: Datum der Belegausgabe)
-2. Start time of the action (DE: Zeitpunkt des Vorgangbeginns)
-3. End time of the action (DE: Zeitpunkt der Vorgangsbeendigung)
-4. In the special case of long lasting actions (DE: lang anhaltender Bestellvorgang): Start time of the first order (DE: Startzeitpunkt der ersten „Bestellung“ im Bondruck) 
+1. Date and time of receipt issuance/creation (DE: Datum der Belegausgabe)
+2. Start date and time of the transaction (DE: Zeitpunkt des Vorgangbeginns)
+3. End date and time of the transaction (DE: Zeitpunkt der Vorgangsbeendigung)
+4. In the special case of long lasting transactions (DE: lang anhaltender Bestellvorgang): Start date and time of the first order (DE: Startzeitpunkt der ersten „Bestellung“ im Bondruck)
 
-The datetime of the receipt creation can be transmitted in the recept request by using the field `ReceiptRequest.cbReceiptMoment`. It must be provided in UTC.
+The time of the receipt creation can be transmitted in the receipt request by using the field `ReceiptRequest.cbReceiptMoment`. It must be provided in UTC.
 
-For a receipt request of type `pos-receipt`, the ftMiddleware returns 3 dates in the signature block that can be used for printing on the receipt:
+For a receipt request of type `pos-receipt`, fiskaltrust.Middleware returns three dates in the signature block that can be used for printing the receipt:
 
-1. time of the start transaction submitted to the TSE (`ftSignatureType`: `0x4445000000000019`).
-2. time of the finish transaction submitted to the TSE (`ftSignatureType`: `0x444500000000001A`).
-3. calculated start time of the action = the minimum of the datetimes (`ReceiptRequest.cbReceiptMoment,ftChargeItem[i].Moment, ftPayItem[i].Moment`) transmitted to the ft.Middleware in the pos-receipt request. The entry has the signature type: `ftSignatureType`: `0x444500000000001F`.
+1. Time of the start transaction submitted to the TSE (`ftSignatureType`: `0x4445000000000019`).
+2. Time of the finish transaction submitted to the TSE (`ftSignatureType`: `0x444500000000001A`).
```

Comment:
```suggestion
2. Time of the finish transaction submitted from the TSE (`ftSignatureType`: `0x444500000000001A`).
```

Reply:
From? Is that value coming **from** the TSE? Probably would use _by_ then.

Reply:
I think these values are coming from the tse. The "by" was my 2nd option.

Reply:
@TSchmiedlechner, are these values submitted to the TSE or do they come back from it?